lib/src/main/java/com/cloudinary/android/preprocess/Parameters.java

```diff
@@ -0,0 +1,15 @@
+package com.cloudinary.android.preprocess;
+
+/**
+ * Video transcoding parameters
+ */
+public class Parameters {
+    public String requestId;
+    public String targetFilePath;
```

Comment:
`targetFilePath` should not be a developer option, it should be removed here and used internally in the preprocess (See comment below).